### ISSUE TITLE
AV: Placeholder image alt text

### ISSUE THREAD

**Issue by 12** [BBC-archive/psammead#1840]
**Is your feature request related to a problem? Please describe.**
We need to pass an empty value to the `alt` tag of the placeholder for the media player. This means that this element will be ignored by screen readers. Currently this is hardcoded to `Image Alt` here:

https://github.com/bbc/psammead/blob/8be9f35aa5012c00998bcd5ddaca3a9f0d6d6549/packages/components/psammead-media-player/src/Placeholder/index.jsx#L15-L19

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Testing notes**
This will be covered by Snapshot tests. We will not include Cypress tests on this ticket.

We will want to do an `a11y` swarm and some minor manual regression.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

**Comment by greenc05** [comment]
What's this for? The screen reader UX doesn't ask for any image alt text for the media player - the alt attribute should be empty, it does however ask for the announced text such as 'Play video, "Video Title", 1 minute 40 seconds' on the play button

**Comment by pjlee11** [comment]
Spoke with @greenc05 IRL, the `alt` value for this particular image should be an empty string as the `<image>` is purely presentational (mimicking the SMP instance) and therefore should be ignored by screen reader. having `alt` as an empty string is valid and achieves the desired behaviour.
